lib/compat/wordpress-6.0/class-wp-webfonts.php

```diff
@@ -16,7 +16,15 @@ class WP_Webfonts {
 	 * @access private
 	 * @var array
 	 */
-	private $webfonts = array();
+	private $registered_webfonts = array();
+
+	/**
+	 * An array of enqueued webfonts.
+	 *
+	 * @access private
```

Comment:
```suggestion
```
Core doesn't use `@access` to identify properties visibility. This annonation can be removed.